microsoft-edge/extensions-chromium/publish/update.md

```diff
@@ -1,62 +1,76 @@
 ---
-description: The process of updating extension on Microsoft Store.
+description: The process of updating or removing extensions from the Microsoft Store
```

Comment:
We use them interchangeably for all our conversations. I think we can maintain both unless we take a hard stance against any at a later stage.

Reply:
I recommend using only 1 term in the docs. I recommend using the following term because it aligns closer with the UI - Microsoft Edge Add-ons store. Can we make the update please?